### ISSUE TITLE
Any plans for a 1.12.2 Release?

### ISSUE THREAD

**Issue by Vyraal1** [Matryoshika/Underworld#22]
I like the look of your mod and was looking to create an overworld clone dimension but underground. Cavern adds too much non-worldgen content, so I would prefer yours if it is accessible for 1.12.2

**Comment by Matryoshika** [comment]
Yes, I am planning to port Underworld to 1.11 & 1.12, but I've hit a wall with worldgen structures right now.
I've been working on adding proper villages to Underworld, and while they do spawn now, they do so ontop of the bedrock ceiling...
With how bad the _"spaghetti-coding"_ of village spawning is in Minecraft (and I'd almost want to add _"Lasagna-coding"_ as well (Look up these phrases for a satirical laugh, if you may)) it's been almost hell trying to move the vertical placement of villages to appropriate levels.

Right now, the most plausible option is "Let villages spawn on top of bedrock. Load a lot of chunks around the center of the village, and find every non-air-and-non-bedrock block, and cut-n-paste to sea-level"...
As you can imagine, that'll be horrible for performance, and the inner server-admin in me is crying at this prospect, but if I want to keep modded structures (IE, forestry, rftools houses etc) that is the best option between "copy ALL the minecraft village code and HIJACK the worldgen using horribly bad HACKY code" and UW's (kinda) motto of "make seamless worldgen seem easy".

Once I find a good solution to the aforementioned issue, heck yeah I'll be updating
